src/components/mfa/mfa-enroll.html

```diff
@@ -0,0 +1,119 @@
+<div class="sp-mfa-enroll-component">
+  <div class="sp-title">Verification Required</div>
+  <div class="sp-loader" rv-show="state | is 'loading'"></div>
+
+  <div rv-show="section | is 'select'">
+    <p>To protect your account from unauthorized access, you'll need to set up another device to verify your identity when logging in.</p>
+    <p>Please choose a method to set up now.</p>
+    <div class="sp-factors">
+      <div class="sp-factor" rv-each-factor="factors" rv-on-click="selectFactor" rv-data-id="factor.id">
+        <h2>{factor.title}</h2>
+        <p>{factor.description}</p>
+      </div>
+    </div>
+  </div>
+
+  <div rv-show="section | is 'setup'">
+    <div rv-show="selectedFactor.id | is 'sms'">
+      <form rv-on-submit="onFormSubmit">
+        <div>
+          <p class="sp-mfa-title">Set up SMS</p>
+          <p>Enter a phone number that can receive SMS text messages. You'll need to have access to this phone when logging in to your account.</p>
+        </div>
+
+        <fieldset class="sp-form-group">
+          <label for="sp-mfa-sms-phone-number">Phone number</label>
+          <input id="sp-mfa-sms-phone-number"
```

Comment:
I fixed this. It was just for being able to have the label linked to the input. But I ended up wrapping the input in the label and removing the id.